commit c5cb4e1a0339844ae3f55ff1dc4a716c28012f05
Author: Paul Crawford <psc@sat.dundee.ac.uk>
Date:   Tue Jun 28 18:08:48 2016 +0100

    Include linux/param.h for EXEC_PAGESIZE definition
    
    Musl does not include linux/param.h whereas glibc does, so it fails
    to build on musl. Patch supplied by Khem Raj <raj.khem@gmail.com>

diff --git a/src/watchdog.c b/src/watchdog.c
index acf6450..486384a 100644
--- a/src/watchdog.c
+++ b/src/watchdog.c
@@ -26,6 +26,9 @@
 #include <sys/param.h>		/* For EXEC_PAGESIZE */
 #include <linux/oom.h>
 #include <linux/watchdog.h>
+#ifdef __linux__
+#include <linux/param.h>
+#endif
 #include <string.h>
 
 #include <libgen.h>
